### PR TITLE
XML removal (part 4)

### PR DESCRIPTION
**Pull request by fgalan**
Part of #1862 

First bunch of 16 case_xml/ test adapted/removed.

@kzangeli 
@crbrox

### CODE REVIEW

File: test/functionalTest/cases/0647_crash_with_compounds/POST_UpdateContextRequest.test [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases/0647_crash_with_compounds/POST_UpdateContextRequest.test]: This (an a couple more of .test files in this PR) are not actually new files but old ones adapted. However, it seem that there has been so many changes and github doesn't detect them as "moved" but "delete file" + "new file".

NTC (just a remark).

File: test/functionalTest/cases_xml/0722_unhardwire_format_cpr/query_forward_in_json.test [hunk 2]
```diff
@@ -1,249 +0,0 @@
-# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases_xml/0722_unhardwire_format_cpr/query_forward_in_json.test]: I understand that all .test under 0722_unhardwire_format_cpr are testing the `notifyFormat` feature to override default format, which is not longer supported as we have only one format (JSON).

File: test/functionalTest/cases_xml/0789_notifyFormat/notifyFormat_ngsi10.test [hunk 3]
```diff
@@ -1,588 +0,0 @@
-# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases_xml/0789_notifyFormat/notifyFormat_ngsi10.test]: I understand that all .test under 0789_notifyFormat are testing the `notifyFormat` feature to override default format, which is not longer supported as we have only one format (JSON).

**Comment by kzangeli** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**